### ISSUE TITLE
Typo? Page 82/ Code Block 2.27: Normal -> Gamma

### ISSUE THREAD

**Issue by kenjioman** [aloctavodia/BAP3#27]
The likelihood used to be a Normal (code block 2.26), but now is inexplicably a Gamma (text was only talking about showing off pymc's coordinates/ dimensions specification of models vs using `shape`).  This is also this way in cell 36 of Chapter 2's code notebook.

**Comment by aloctavodia** [comment]
You are right, there is a missing paragraph here. This should have been explained as a two step process. First I showed a normal to connect to the previous model and show that we "just" add shapes/dimensions and second I wanted to change the likelihood to a Gamma.
